treeherder/utils/taskcluster_lib_scopes.py

```diff
@@ -0,0 +1,30 @@
+"""
+TODO: Extract this module into a dedicated PyPI package, acting as the
+    Python variant of https://github.com/taskcluster/taskcluster-lib-scopes
+"""
```

Comment:
why does this file exist when there is no code?  Have you already done the TODO?

Reply:
> why does this file exist when there is no code?

I'm not sure I follow. This module does define 2 functions, namely `satisfiesExpression()` & `patternMatch()`.

> Have you already done the TODO?

No, I haven't. This TODO is just a reminder for when we'll have the time to define the Python equivalent for `taskcluster-lib-scopes`. Its priority is low, as doing this gives us a pretty small benefit, but requires a hefty amount of effort.

Reply:
oh, this is the github review interface and I thought there was a file break and instead it was a resolved conversation. sorry for the confusion